### PR TITLE
Add only first option

### PR DESCRIPTION
**Pull request by afdev82**
Give the possibility to stay on the same website when hit a not whitelisted, excluded page or when using the skip_geo parameter and after that visit a url that would be redirected.

I haven't updated the documentation yet, because I wanted to know before what you think about that.

### CODE REVIEW

File: lib/geo_redirect/middleware.rb [hunk 1]
```diff
@@ -61,9 +62,13 @@ def force_redirect?
...
       url = URI.parse(@request.url)
-      query_includes_skip_geo?(url) ||
-        path_not_whitelisted?(url) ||
-        path_blacklisted?(url)
+      host = host_by_hostname(url.host)
+
+      return false unless query_includes_skip_geo?(url) ||
+                          path_not_whitelisted?(url) ||
+                          path_blacklisted?(url)
+      remember_host(host) if @only_first
```
Comment (n0nick) [inline lib/geo_redirect/middleware.rb]: I would not add this side-effect to `skip_redirect?` which is supposed to be a very simple predicate. I would add the logic in an outer layer

**Comment by n0nick** [comment]
@afdev82 thx for the contribution! :)

For this to get merged, please 
1. Rename `only_first` to something more self-explanatory
2. See my inline comment
3. Check out the linter error in Travis (should be resolved by my inline suggestion)
4. Add a test case for the new feature

If you need help with any of these and/or have any questions, let me know.

**Comment by afdev82** [comment]
About the name, it's difficult to find a concise name for this feature (at least for me).
What do you think about `redirect_after_skip` ? (then the meaning will be the opposite of the current value)

**Comment by n0nick** [comment]
how about `redirect_later`? :)

**Comment by afdev82** [comment]
For me it's ok

**Comment by afdev82** [comment]
I tried to fix the rubocop compliant, but I think the minimum number of lines would be 175.

**Comment by n0nick** [comment]
@afdev82 that would be fineI think. The current config is 170 so 175 is a fine stretch :)

Let's get the tests passing and then, if possible, squash everything to a single commit for the feature :+1:

**Comment by afdev82** [comment]
See #27

**Pull request closed without merging**